_data/sites.json

```
@@ -15377,6 +15398,17 @@
         ]
     },
 
+    {
+        "name": "Pottermore",
+        "url": "https://privacyportal.onetrust.com/webform/1b21e05d-c206-4e0b-970e-2d73a23e42e8/780a7716-f409-47f4-afdf-fbe274737e6f",
+        "url_us": "https://privacy.wizardingworld.com/delete-data-request/",
+        "difficulty": "hard",
+        "notes": "Fill out form and submit. Non-US form entries will receive an email to confirm the deletion request. Has United States specific form.",
```

```suggestion
        "difficulty": "hard",
        "notes": "Fill out form and submit. Non-US form entries will receive an email to confirm the deletion request. If you're from the USA, use [this form](https://privacy.wizardingworld.com/delete-data-request/)",
```

There is no country code, but rather language codes, to use URL and notes that are localised to that given language. I think the best solution for the USA form is this one.

---

Thanks for the feedback, that makes sense. Resolved with latest push.